language-server/modules/langserver-core/src/test/resources/completion/new/expression_context/config/anon_func_expr_ctx_config5a.json

```diff
@@ -6,12 +6,393 @@
   "source": "expression_context/source/anon_func_expr_ctx_source5a.bal",
   "items": [
     {
-      "label": "returns",
+      "label": "module1",
+      "kind": "Module",
```

Comment:
Test case has been broken and in this context completion should only contain `returns` keyword